org.eclipse.scout.rt.rest/src/main/java/org/eclipse/scout/rt/rest/chunked/IChunkedDataWriter.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2010, 2025 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.scout.rt.rest.chunked;
+
+import java.io.IOException;
+
+import jakarta.ws.rs.core.GenericType;
+import jakarta.ws.rs.core.Response;
+
+import org.eclipse.scout.rt.platform.BEANS;
+import org.eclipse.scout.rt.platform.Bean;
+
+/**
+ * Writer for a stream of chunked data of type {@code T}.
+ */
+@Bean
+public interface IChunkedDataWriter<T> {
+
+  /**
+   * @return IChunkedDataWriter for chunks of type {@code type}, using a default delimiter.
+   */
+  static <T> IChunkedDataWriter<T> create(Class<T> type) {
+    return create(type, null);
+  }
+
+  /**
+   * @return IChunkedDataWriter for chunks of type {@code type}, using given {@code delimiter}.
+   */
+  static <T> IChunkedDataWriter<T> create(Class<T> type, String delimiter) {
+    //noinspection unchecked
+    return BEANS.get(IChunkedDataWriter.class).init(type, delimiter);
+  }
+
+  /**
+   * @return Initialized IChunkedDataWriter for chunks of type {@code type}, using given {@code delimiter}.
```

Comment:
```suggestion
   * @return initialized IChunkedDataWriter for chunks of type {@code type}, using given {@code delimiter}.
```

Does this return itself or a different instance?

Reply:
Comment updated. Returns this (e.g. the same) instance.